config/environments/production.rb

```diff
@@ -98,4 +98,6 @@
     compress: true,
     compression_min_size: 524_288
   }
+
+  config.middleware.use Redirector
```

Comment:
I think this means the order of the middleware will be slightly different. Does that matter?